### DISCUSSION TITLE
Unable to "require" macros from a file at a different directory by absolute path.

### DISCUSSION THREAD

**Discussion by Priezt** [hylang/hy#2618] [Q&A] [answered]
When using "require", I can only load macros from a file in the same directory.
Is it possible to load from a file at a different directory by absolute path?

For now, I have to use symbol link to simulate loading macros from a different directory.

**Comment by Kodiologist** [comment, marked as answer]
> When using "require", I can only load macros from a file in the same directory.

You should be able to require a file from anywhere in `sys.path`, with the same rules as `import`. If not, it's a bug.

> Is it possible to load from a file at a different directory by absolute path?

If it's not already in `sys.path`, you can temporarily add its parent directory.
